source/manual/github-enterprise.html.md

```diff
@@ -1,11 +0,0 @@
----
```

Comment:
The goal of this page was to record somewhere that we used `github.gds` at some point, in case there are old emails/trello tickets/github PR mentioning it.

Reply:
Restored.